maintenance/projects/js-toolkit/CONTRIBUTING.md

```diff
@@ -70,6 +76,87 @@ This command will download all the dependencies needed by the QA projects contai
 
 Note that the `link-js-toolkit` will move all JS Toolkit dependencies in the QA projects to a `link-js-toolkit` section in the `package.json`. This is to prevent Yarn from trying to download these packages from npmjs.com.
 
+### Testing your local version of JS Toolkit
+
+The normal way to use the JS Toolkit in your projects is by letting `npm` download it from npmjs.com.
+
+However, for that to happen, a version must be released and there are times when you simply want to test what you have modified locally in a clone of this repo. In that case, the setup is a bit more difficult because you need to tell `yarn` to use the local versions of the JS Toolkit packages.
+
+But... we have `link-js-toolkit` for that.
+
+The procedure to setup a `yarn` workspace that uses your local copy of the JS Toolkit is as follows:
+
+1. Create a directory (f.e: `toolkit-workspace`) and run `yarn init -y` to create a `package.json` file on it.
+2. Edit the `package.json` file to add the following:
+
+```json
+{
+	"private": true,
+	"workspaces": {
+		"packages": ["packages/*"]
+	}
+}
+```
+
+This is to create a yarn workspace.
+
+3. Create the `packages` directory inside `toolkit-worspace`.
+4. Run `yarn` to create the `yarn.lock` file.
+5. Go to `liferay-frontend-projects/maintenance/projects/js-toolkit/resources/devtools/link-js-toolkit` folder and run `yarn link` inside. That will make `link-js-toolkit` available as a command in your machine.
+6. Run `link-js-toolkit -p`. This will run `yarn link` inside each process of the JS Toolkit, registering your local project folders with `yarn` so that it then knows where to find them. You can see all projects registered with `yarn link` in your `~/.config/yarn/link` folder (it contains symlinks to your local projects). If you want to unlink a project simply follow the link in that folder to get to your local project folder and run `yarn unlink` there. Refer to [yarn link's documentation](https://classic.yarnpkg.com/en/docs/cli/link/) for more details.
```

Comment:
"inside each process" → "inside each package"

Reply:
True. However I use `project` for the directory + source code and package for the artifact we download from npmjs.com.

Reply:
But what does "process" actually mean? To me it sounds like a running executable.

Reply:
Process was incorrect, as you said :-) (I use process for running executable, too). I was proposing:

"inside each process" → "inside each project"

better than:

"inside each process" → "inside each package"

Because I use project for the source folder and package when it is used via node_modules in another place.

Of course one thing is what I use and another completely different what the reader may interpret in the end, but I try... :sweat_smile:

Reply:
> but I try...

`try {`
This will run `yarn link` inside each project of the JS Toolkit
`}
catch {`
This will run `yarn link` inside each directory/package/place/thing of the JS Toolkit
`}`

Reply:
This will run :running_woman:

Reply:
![run](https://media.giphy.com/media/2bUpP71bbVnZ3x7lgQ/giphy.gif)